.github/maintainers_guide.md

```diff
@@ -19,30 +19,58 @@ Test code should be written in syntax that runs on the oldest supported Node.js
 
 A useful trick for debugging inside tests is to use the Chrome Debugging Protocol feature of Node.js to set breakpoints and interactively debug. In order to do this you must run mocha directly. This means that you should have already linted the source (`npm run lint`), manually. You then run the tests using the following command: `./node_modules/.bin/mocha test/{test-name}.js --debug-brk --inspect` (replace {test-name} with an actual test file).
 
-### Generating Documentation
+### Managing Documentation
```

Comment:
I'd probably simply link to the docs/README.md from here instead of duplicating most of the content.